app/src/main/res/drawable/item_row_divider_decoration.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <size android:height="1dp" />
+    <solid android:color="@color/colorDivider" />
```

Comment:
This is in the color file so it doesn't need to have color in its value. It can just be divider.